package/yast2-iscsi-client.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Jun 11 10:00:09 CEST 2015 - gs@suse.de
+
+- 2.17.42
+- add Requires: open-iscsi to sepc-file (bnc #932758)
```

Comment:
On the other hand it doesn't say why ... and I think this is more important, because in two years we will be looking for `why is it there?`
Maybe better in comment above `Requires` in spec file